app/serializers/service_provider_serializer.rb

```diff
@@ -0,0 +1,38 @@
+class ServiceProviderSerializer < ActiveModel::Serializer
+  attributes(
```

Comment:
the parser wants parens if they're on different lines, this would be acceptable, but it's neither here nor there:

```ruby
attributes :foo,
           :bar
```